mmv1/third_party/terraform/.goreleaser.yml.erb

```diff
@@ -30,7 +30,7 @@ builds:
       - goarch: arm64
         goos: windows
     ldflags:
-      - -s -w -X internal/provider.Version={{.Version}}
+      - -s -w -X internal/provider.ProviderVersion={{.Version}}
```

Comment:
It looks like the user agent version is pulled `ProviderVersion` set in `version/version.go` in each provider - does this end up setting that build variable somehow?

Reply:
Right, that is the provider version that the user agent uses. If i'm understanding your question correctly, [yes](https://github.com/hashicorp/terraform-provider-google/blob/main/version/version.go#L4).

When we updated the release process to use goreleaser, I had copied the goreleaser code from a different provider, who defined the variable `Version` in their `version.go` rather than `ProviderVersion`, which I didn't notice at the time, and that's why `ProviderVersion` hasn't been updated since.

Reply:
cool, yeah, just wanted to double-check that it gets translated to `version.ProviderVersion` at some point. Thanks!